Hunting Queries/DnsEvents/DNS_HighPercentNXDomainCount.yaml

```diff
@@ -2,7 +2,7 @@ id: 543e1ec6-ee5e-4368-aaa6-405f0551ba5c
 name: DNS high percentage NXDomain count
```

Comment:
Do we want to rename this with DGA like you did for the Detection version of it?